torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1351,12 +1357,14 @@ def repr_symbol_to_source():
             f"not in {repr_symbol_to_source()}.  If this assert is failing, it could be "
             "due to the issue described in https://github.com/pytorch/pytorch/pull/90665"
         )
-        return self.source_ref(self.symbol_to_source[expr][0])
+        source = self.symbol_to_source[expr][0]
+        self.sources.append(source)
```

Comment:
This seems weird

Reply:
Why?

Reply:
I think this is subsumed by my source_ref comment